user/caching.md

```diff
@@ -147,7 +147,7 @@ cache:
   - $HOME/.m2
 ```
 
-As you can see, it is also possible to use environment variables in the directories.
+As you can see, it is also possible to use environment variables in the directories. Paths that do not start with a `/` are relative to `$TRAVIS_BUILD_DIR`. When, after possible variable expansion, the path starts with a `/`, that absolute path is used. Please be aware that the `travis` user needs to have write permissions in this directory.
```

Comment:
And here maybe something like:

After variable expansion, paths that

> * do *not* start with a `/` are relative to `$TRAVIS_BUILD_DIR`
> * do start with a `/` are absolute

To create the cache, the `travis` user needs to have write permission to the directory.

(I'd say *to* not *in* -- what do you think?)